### DISCUSSION TITLE
Multiple Gerrit repos migrate into GHES

### DISCUSSION THREAD

**Discussion by Hkri-se** [community/community#136931] [Enterprise]
### Select Topic Area

Question

### Body

Hi,

Hi Nick,

We have new requirement from Gerrit to GHES code migration. Below are the key points for the migration.
-	We have multiple folders, under the folders having multiple repos.
-	Our requirement is multiple repos needs to migrate into GHES as a single repo.
-	Along with this need to migrate all branches, tags and commit history.

Could you please help me, is this migration process is it possible from Gerrit to GHES as mention the above requirements.

Thanks,
Harikrishna CH.

**Comment by queenofcorgis** [comment]
Hi @SESA647088 - Thanks for posting this. We are looking into supporting Gerrit "Topics" and I've noted your interest as well. However, we don't have a specific timeline or further details to share. We will keep you updated!
